packages/store/tests/action.spec.ts

```diff
@@ -408,6 +408,34 @@ describe('Action', () => {
           tick(1);
           expect(actionStatus).toEqual([true, false]);
         }));
+
+        it('should be executing when action is dispatched multiple times', fakeAsync(() => {
+          const actionStatus: boolean[] = [];
+
+          actions.pipe(ofActionExecuting(AsyncAction1)).subscribe(executing => {
+            actionStatus.push(executing);
+          });
+
+          store.dispatch(new AsyncAction1());
+          expect(actionStatus).toEqual([true]);
+          store.dispatch(new AsyncAction1());
+          tick(1);
+          expect(actionStatus).toEqual([true, true, false, false]);
```

Comment:
This test shows that the bug that I suspected is actually happening.
It should only become true when both actions have completed.
I would expect this expectation to rather be `true, true, true, false`

Reply:
you are right, i ended up adding code for this scenario check last commit please (wasn't able to do it with less code)

thanks!